ecmd-core/pyecmd/generate_pyecmd.py

```diff
@@ -1,5 +1,4 @@
-#!/usr/bin/python
-from __future__ import print_function
+#!/usr/bin/python3
```

Comment:
@fenkes-ibm can this change to
```
#!/usr/bin/env python3
```
@kahnevan we'll need to make sure python3 is in our environment when building.